app/main/services/query_builder.py

```diff
@@ -61,14 +61,13 @@ def highlight_clause(mapping):
         "pre_tags": ["<mark class='search-result-highlighted-text'>"],
         "post_tags": ["</mark>"]
     }
-    highlights["fields"] = {}
-
     # Get all fields searched and allow non-matches to a max of the searchSummary limit
-    for field in mapping.fields_by_prefix.get(mapping.text_search_field_prefix, ()):
-        highlights["fields"]["_".join((mapping.text_search_field_prefix, field))] = {
+    highlights["fields"] = {
+        f"{mapping.text_search_field_prefix}_*": {
             "number_of_fragments": 0,
-            "no_match_size": 500
+            "no_match_size": 500,
         }
+    }
 
```

Comment:
Might as well suggest this here instead of merging my PR but these can be chucked in the root of the highlighting options like:

```
    highlights = {
        "fields": {f"{mapping.text_search_field_prefix}_*": {}},
        "number_of_fragments": 0,
        "no_match_size": 500,
        "encoder": "html",
        "pre_tags": ["<mark class='search-result-highlighted-text'>"],
        "post_tags": ["</mark>"],
    }
```

https://www.elastic.co/guide/en/elasticsearch/reference/master/search-request-highlighting.html#override-global-settings

Up to you if you want to do it here or I can do it in a separate PR after this is merged

Reply:
Doing this